microservices/payment/src/entities/customer.ts

```diff
@@ -1,31 +1,42 @@
 import { IsUndefinable } from '@lomray/microservice-helpers';
-import { Allow, Length, IsObject } from 'class-validator';
+import { Length, IsObject, IsString } from 'class-validator';
 import { JSONSchema } from 'class-validator-jsonschema';
-import { Column, Entity, Index, PrimaryColumn } from 'typeorm';
+import { Column, Entity, Index, OneToMany, PrimaryColumn, Unique } from 'typeorm';
+import Transaction from '@entities/transaction';
 
 export interface ICustomerParams {
   accountId?: string;
 }
 
+@JSONSchema({
+  properties: {
+    transactions: { $ref: '#/definitions/Transactions', type: 'array' },
+  },
+})
 @Entity()
 class Customer {
   @PrimaryColumn({ type: 'varchar', length: 18 })
-  @Allow()
+  @IsString()
+  @Length(1, 18)
```

Comment:
```suggestion
  @Length(1, 18)
```

Length - enough

fix other fields